### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2023-12-28

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#122] [General]
Previous agenda: #117 

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
    - [x] https://github.com/space-ros/process_sarif/pull/9
    - [x] https://github.com/space-ros/space-ros/pull/116
    - [x] https://github.com/space-ros/space-ros/pull/123
- [x] Review issues and PRs ([nominated for next release](https://github.com/orgs/space-ros/projects/3/views/6)).
- [x] Open next meeting discussion

**Comment by Bckempa** [comment]
- [x] Triage (briefly) #120 and #121
I don't expect to address these before the next release, but they should be on someone's (probably @EzraBrooks's?) radar

**Comment by Bckempa** [comment]
- [x] CI Miss: Removal of old branches (#97) broke building the main Space ROS docker container because of a bug in the Earthfile which pinned the source of the repos file to the `earthly-wrapper` branch (reported by @yuyuqq in https://github.com/space-ros/docker/issues/94). This will be fixed by https://github.com/space-ros/docker/pull/90 soon. CI didn't catch this because it had no reason to build. On one hand that build **should** only be using pinned versions and not behave differently on builds without code changes, on the other a scheduled rebuild would have alerted us that this wasn't the case. Should we add a scheduled build to the [`space-ros/docker` earthly-build workflow](https://github.com/space-ros/docker/actions/workflows/earthly-build.yaml)?

**Reply by ivanperez-keera** [reply]
We just discussed it but to keep this documented: I just pushed a branch `earthly-wrapper` that is a fork of the current `main`, to keep the CI job working until we push a more permanent solution.

**Comment by ivanperez-keera** [comment]
Another issue to discuss: IKOS 3.2 has been released.

We could 1) try it in the docker image, 2) if it works, pin that version, and 3) check that it's integrated in the dashboard (assuming version 3.1 is).

There's also https://github.com/space-ros/space-ros/issues/29, which I don't know if it will survive.

**Reply by ivanperez-keera** [reply]
We made a decision to 1) investigate if this is really running currently and, if so, 2) update the version being used in Space ROS (pin version), and send a PR with that change.

**Comment by Bckempa** [comment]
Next Agenda posted at https://github.com/space-ros/space-ros/discussions/124
